readthedocs/projects/models.py

```diff
@@ -114,6 +113,16 @@ class Project(models.Model):
         help_text=_("Google Analytics Tracking ID (ex. UA-22345342-1). "
                     "This may slow down your page loads."))
 
+    # Sphinx specific build options.
```

Comment:
I'm happy to keep this PR as is, and address the naming stuff in a v2 where we move it into a completely separate model, or into a YAML file, as well, if it's a lot of work to rename all this stuff.